File: applications/event-account-jdbc-sink/src/main/java/com/vmware/account/jdbc/sink/domain/LocationEntity.java
package com.vmware.account.jdbc.sink.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "evt_locations",schema="evt_showcase")
@Data
@NoArgsConstructor
public class LocationEntity {
    @Id
    private String id;
    private String address;
    private String cityTown;
    private String stateProvince;
    private String zipPostalCode;
    private String countryCode;
}

File: applications/account-jdbc-amqp-sink/src/main/kotlin/com/vmware/tanzu/data/services/rabbitmq/streaming/account/domain/LocationEntity.kt
package com.vmware.tanzu.data.services.rabbitmq.streaming.account.domain

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table


@Entity
@Table(name = "evt_locations")
data class LocationEntity(
    @Id var id: String = "",
    var address: String= "",
    var cityTown: String = "",
    var stateProvince: String = "",
    var zipPostalCode: String = "",
    var countryCode : String = ""
)
